gobblin-cluster/build.gradle

```diff
@@ -0,0 +1,99 @@
+// (c) 2014 LinkedIn Corp. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied.
+//
+
+apply plugin: 'java'
+
+dependencies {
+  compile project(":gobblin-admin")
+  compile project(":gobblin-api")
+  compile project(":gobblin-core")
+  compile project(":gobblin-metrics")
+  compile project(":gobblin-metastore")
+  compile project(":gobblin-runtime")
+  compile project(":gobblin-utility")
+  compile project(path: ':gobblin-rest-service:gobblin-rest-api', configuration: 'restClient')
+  compile project(":gobblin-rest-service:gobblin-rest-server")
+
+  compile externalDependency.avro
+  compile externalDependency.commonsConfiguration
+  compile externalDependency.quartz
+  compile externalDependency.guava
+  compile externalDependency.commonsLang
+  compile externalDependency.slf4j
+  compile externalDependency.log4j
+  compile externalDependency.commonsCli
+  compile externalDependency.gson
+  compile externalDependency.hiveCommon
```

Comment:
yes, from gobblin-yarn, so we also have a few Yarn related dependencies (which would be removed as we develop, since this should not eventually depend on Yarn)